deploy/jiva-csi.yaml

```diff
@@ -35,7 +35,7 @@ metadata:
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
-  name: openebs-csi-controller-critical
+  name: openebs-jiva-csi-controller-critical
```

Comment:
@shovanmaity - can you make the changes here as well please. 
https://github.com/openebs/jiva-operator/blob/develop/deploy/jiva-csi.yaml#L38